deploy/charts/monitoring/dashboards/central.json

```diff
@@ -238,20 +237,202 @@
       "title": "Heap Memory Usage",
       "type": "timeseries"
     },
+    {
+      "datasource": null,
+      "description": "Pod Memory Usage",
```

Comment:
I'd suggest to set the Grafana unit to megabytes. In the screenshot it looks like bytes / unit less.

Reply:
Done